<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FigmaDisplayer</class>
 <widget class="QMainWindow" name="FigmaDisplayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FigmaDisplayer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_project_name">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>670</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTipDuration">
     <number>-7</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_access_token">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>710</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_project_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Project name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_access_token">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Access token</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_apply">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>690</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>apply</string>
    </property>
   </widget>
   <widget class="QLabel" name="canvas">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>980</width>
      <height>650</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
